dcos/dcos-ui
# DCOS_OSS-4502: service config dropdown content overflows

## Testing
1. In `plugins/services/src/js/containers/service-configuration/ServiceConfiguration.js`, on line 143, change the `<DateFormat />` component to just be a span with a long string
2. Open a service from the "Services" tab
3. Go to the "Configuration" tab
4. Click the dropdown at the top of that screen

The long string you replaced `<DateFormat />` with should not overflow the container - it should appear truncated

## Trade-offs
None

## Dependencies
None

## Screenshots
Before:
<img width="942" alt="screen shot 2018-12-10 at 3 42 01 pm" src="https://user-images.githubusercontent.com/2313998/49761713-d5143c80-fc95-11e8-888d-61faea45bd54.png">

After:
<img width="921" alt="screen shot 2018-12-10 at 3 45 04 pm" src="https://user-images.githubusercontent.com/2313998/49761725-d9405a00-fc95-11e8-9160-31b623b3fa01.png">

Note: I tested this in Chrome, FF, Safari, and Edge
